src/main/java/edu/harvard/iq/dataverse/Template.java

```diff
@@ -256,26 +268,31 @@ public void setMetadataValueBlocks() {
         metadataBlocksForView.clear();
         metadataBlocksForEdit.clear();
         List<DatasetField> filledInFields = this.getDatasetFields(); 
+
+        Map<String, String> instructionsMap = getInstructionsMap();
         
         List <MetadataBlock> viewMDB = new ArrayList<>();
-        List <MetadataBlock> editMDB=this.getDataverse().getMetadataBlocks(true);
+        List <MetadataBlock> editMDB=this.getDataverse().getMetadataBlocks(false);
             
+        //The metadatablocks in this template include any from the Dataverse it is associated with 
+        //plus any others where the template has a displayable field (i.e. from before a block was dropped in the dataverse/collection)
         viewMDB.addAll(this.getDataverse().getMetadataBlocks(true));
```

Comment:
My guess is that this call is no longer needed. As is it gets no blocks unless the current one is the metadatablockroot and it is the loop below that adds the right blocks. Looking back at #8837 where this was added, there used to be one list of blocks for edit and view, which is why getting the available blocks was needed. Now that this variable is just for view, I don't think getting the possible blocks is needed. (I also don't think it hurts in that for view, only the blocks with defined fields appear anyway, so having extra blocks in viewMDB doesn't break anything.